stdlib/termios.pyi

```diff
@@ -3,10 +3,12 @@ from _typeshed import FileDescriptorLike
 from typing import Any
 from typing_extensions import TypeAlias
 
-if sys.platform != "win32":
-    # Must be a list of length 7, containing 6 ints and a list of NCCS 1-character bytes or ints.
-    _Attr: TypeAlias = list[int | list[bytes | int]]
+# Must be a list of length 7, containing 6 ints and a list of NCCS 1-character bytes or ints.
+_Attr: TypeAlias = list[int | list[bytes | int]]
```

Comment:
To avoid issues with invariance...

```suggestion
_Attr: TypeAlias = list[int] | list[list[bytes]] | list[int | list[bytes]] | list[list[int]] | list[int | list[int]] | list[list[int | bytes]] | list[int | list[int | bytes]]
```

Reply:
Good catch, although `list[int]` and `list[list[...]]` are not valid as this is a fixed-size list with specific types for specific positions.